README.md

```diff
@@ -9,16 +9,13 @@ This repository hosts the CSI KubeVirt driver and all of its build and dependent
 
 ## Deployment
 //TODO WIP
-- create a service-account with name on the kubevirt cluster (infra-cluster)
-- set RBAC rules for the service account (//TODO supply file, doc)
-- designate a namespace to deploy the VMs for the tenant cluster nodes
-- deploy tenant cluster
-- deploy files under `deploy/*`
-- edit the secret `infra-cluster-credentials` in namespace `kubevirt-csi-driver`
-    - set apiUrl: [base64 of https://<infra-cluster-api-url>]  
-    - set service-ca.crt : [base64 of <infra-cluster serivce-ca.crt>] (copy service-ca.crt value from infra-cluster-sa secret)
-    - set namespace: [base64 of <infra-cluster namespace>]
-    - set token : [base64 of <infra-cluser token>] (copy token value from infra-cluster-sa secret )
+- use `deploy/infra-cluster-service-account.yaml` to create a service account in kubevirt cluster (replace INFRA_NAMESPACE with the kubevirt cluster namepsace)
+- use `deploy/secret.yaml` for creating the necessary secret in the tenant cluster
+    - set apiUrl: [base64 of infra cluster API URL. E.g. base64 of https://cluster.mydomain.co:6443]
+    - set service-ca.crt : copy value from infra service account created in previous section
```

Comment:
the value is on the secret of the service account, not the service account itself